importer/MtDe.py

```diff
@@ -0,0 +1,96 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class MtDe(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Use the 'gemeinde' field as
+        municipality.
+        """
+        municip_raw = self.gemeinde
+        municip_q = utils.q_from_wikipedia("de", municip_raw)
+        if municip_q:
+            self.add_statement("located_adm", municip_q)
+        else:
+            self.add_to_report("gemeinde", self.gemeinde, "located_adm")
```

Comment:
Is it an idea to fall back on `region-iso` for the ones where this fails? (in addition to report)

Reply:
Added a check.